pkgs/top-level/perl-packages.nix

```diff
@@ -4846,6 +4846,26 @@ with self; {
     };
   };
 
+  CryptHSXKPasswd = buildPerlPackage {
+    pname = "Crypt-HSXKPasswd";
+    version = "3.6";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/B/BA/BARTB/Crypt-HSXKPasswd-v3.6.tar.gz";
+      hash = "sha256-lZ3MX58BG/ALha0i31ZrerK/XqHTYrDeD7WuKfvEWLM=";
+    };
+    buildInputs = [ Clone DateTime FileHomeDir FileShare FileShareDir GetoptLong JSON ListMoreUtils MathRound Readonly TextUnidecode TypeTiny ];
```

Comment:
Caught this late, but I think this should be `propagatedBuildInputs` - otherwise the main program gets broken.

Also needs some extra work for running properly on darwin - will prepare a small fix PR.

Reply:
Thanks for noticing and fixing this, this has been the first time I've came across `propagatedBuildInputs` (although looking at some of the other Perl packages I am not sure how I missed it the first time round).

Reply:
No worries - thanks too for packaging this great software! 🎉